src/boot/efi/vmm.c

```diff
@@ -307,9 +307,10 @@ const char* smbios_find_oem_string(const char *name) {
 
         assert(left >= type11->header.length); /* get_smbios_table() already validated this */
         left -= type11->header.length;
+        const char *limit = s + left;
```

Comment:
Nit, could be moved in the `for(` initialization field so that it's scoped? Doesn't matter that much